tests/test_explorer.py

```diff
@@ -249,11 +265,11 @@ def test_reload_button_clicked(self, mocked_load_file_tree):
         # Once when the app is created, once explicitly.
         self.assertEqual(mocked_load_file_tree.call_count, 2)
 
-    @mock.patch("iquip.apps.explorer.ExplorerApp.openExperiment")
-    def test_open_button_clicked(self, mocked_open_experiment):
+    @mock.patch("iquip.apps.explorer.ExplorerApp.openButtonClicked")
+    def test_open_button_clicked(self, mocked_open_button_clicked):
```

Comment:
Well, this test method name is duplicate, but if you think it's fine, then no problem.

Reply:
They are located in different test classes, a unit test class and a functional test class, hence I think it is okay👍